tests/integration/testlib/assertlib.py

```diff
@@ -53,3 +53,8 @@ def assert_absent(*ifnames):
 
     current_state = statelib.show_only(ifnames)
     assert not current_state[INTERFACES]
+
+
+def assert_all_equal(iface_macs):
```

Comment:
IMHO the `iface_` prefix does not add more information to the function. Also this function is not specific to macs, so the variable could just be called `items`.